controls/5.04-node.rb

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 The inspec-gke-cis-benchmark Authors
```

Comment:
the filename should probably be 5.04-node-metadata to easier distinguish from 5.5